app/views/templates/ssh/check_update.erb

```diff
@@ -0,0 +1,33 @@
+<%#
+kind: job_template
+name: Check Update - SSH Default
+model: JobTemplate
+job_category: Packages
+description_format: "Check for package updates"
+provider_type: SSH
+%>
+
+<%
+  supported_families = ['Redhat', 'Debian', 'Suse']
+  render_error(N_('Unsupported or no operating system found for this host.')) unless @host.operatingsystem && supported_families.include?(@host.operatingsystem.family)
+
+  command = case @host.operatingsystem.family
+            when 'Redhat'
+              'yum check-update'
+            when 'Debian'
+              'apt list --upgradable'
+            when 'Suse'
+              'zypper list-updates'
+            end
+-%>
+
+<%= command %>
+<% if command.start_with? 'yum' %>
+rc=$?
+# yum check-update returns 110 even on success
+if [ $rc -eq 110 ]; then
+    (exit 0)
+else
+    (exit $rc)
+fi
```

Comment:
Yum on my system returns "100", man page says it means there are updates available. I can't find any info about 110, can you elaborate and link docs?

What the script is supposed to return? Can you explain here and also in the description field or at least in ERB comment for future edits?

Reply:
This must have been a typo.

Remote execution considers jobs which ended with exit code 0 as successful and all others as failed. In here we override the exit code to 0 in case yum check-update returns 100 so the job is marked as successful (because there was no error) in remote execution.